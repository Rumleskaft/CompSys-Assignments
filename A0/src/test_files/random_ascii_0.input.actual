test_files/random_ascii_0.input: ASCII text
