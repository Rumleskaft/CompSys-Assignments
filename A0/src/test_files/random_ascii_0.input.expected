test_files/random_ascii_0.input: very short file (no magic)
